### PR TITLE
Add data-integration for widget

### PR DESCRIPTION
**Pull request by bautrukevich**

### CODE REVIEW

**Comment by bautrukevich** [comment]
[Widget & wordpress: обновить](https://trello.com/c/zDFMjIxo/217-widget-wordpress-%D0%BE%D0%B1%D0%BD%D0%BE%D0%B2%D0%B8%D1%82%D1%8C)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,145 +1,171 @@
...
-### 2.6.0 ([Download](http://downloads.wordpress.org/plugin/uploadcare.2.6.0.zip))
-- Updated [https://github.com/uploadcare/uploadcare-php](https://github.com/uploadcare/uploadcare-php)
-- Added integration to the widget config to provide info
-  on the Wordpress version and plugin version for User Agent with requests
-  to Uploadcare Upload API.
+The format is based now on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-### 2.5.3 ([Download](http://downloads.wordpress.org/plugin/uploadcare.2.5.3.zip))
+## [2.2.0] — 2018-05-14
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: oopsie :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,145 +1,171 @@
...
+
+### Added
+- Integration to the widget config to provide info on the Wordpress version and plugin version for User Agent with requests to Uploadcare Upload API.
+
+### Changed
+- Updated [uploadcare/uploadcare-php](https://github.com/uploadcare/uploadcare-php) to 2.2.0 version
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: to 2.2.0 version

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,145 +1,171 @@
...
-  on the Wordpress version and plugin version for User Agent with requests
-  to Uploadcare Upload API.
+The format is based now on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-### 2.5.3 ([Download](http://downloads.wordpress.org/plugin/uploadcare.2.5.3.zip))
+## [2.2.0] — 2018-05-14
+
+### Added
+- Integration to the widget config to provide info on the Wordpress version and plugin version for User Agent with requests to Uploadcare Upload API.
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: move to changed:

updated User Agent header compilation

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,139 +1,179 @@
...
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [2.6.1] — 2018-05-14
+
+### Changed
+- Updated [uploadcare/uploadcare-php](https://github.com/uploadcare/uploadcare-php) to 2.2.1 version
+
+### ([Download](http://downloads.wordpress.org/plugin/uploadcare.2.6.1.zip))
+
+## [2.6.0] — 2018-05-14
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: we don't need 2.6.1 yet, just update underlying php lib within 2.6.0

File: inc/utils.php [hunk 5]
```diff
@@ -6,12 +6,14 @@
...
-    $user_agent = 'Uploadcare Wordpress ' . UPLOADCARE_PLUGIN_VERSION . '/' . $wp_version;
-    return new Uploadcare\Api(
+    $api = new Uploadcare\Api(
         get_option('uploadcare_public'),
-        get_option('uploadcare_secret'),
-        $user_agent
+        get_option('uploadcare_secret')
     );
+    $api->setFramework('Wordpress', $wp_version);
+    $api->setExtension('PHPUploadcare-Wordpress', UPLOADCARE_PLUGIN_VERSION);
```
[the remark this answers is not in the document]
Comment (dmitry-mukhin) [reply inc/utils.php]: ✅

File: uploadcare.php [hunk 6]
```diff
@@ -3,7 +3,7 @@
 Plugin Name: Uploadcare
 Plugin URI: http://github.com/uploadcare/uploadcare-wordpress
 Description: Uploadcare let's you upload anything from anywhere (Instagram, Facebook, Dropbox, etc.)
-Version: 2.5.3
+Version: 2.6.0
```
Comment (dmitry-mukhin) [inline uploadcare.php]: accidentally, this is what we want.

File: uploadcare.php [hunk 7]
```diff
@@ -14,7 +14,7 @@
     exit("Uploadcare plugin requires PHP version <b>5.3+</b>, you've got <b>" . PHP_VERSION . "</b>");
 }
 
-define('UPLOADCARE_PLUGIN_VERSION', '2.5.3');
+define('UPLOADCARE_PLUGIN_VERSION', '2.6.1');
```
Comment (dmitry-mukhin) [inline uploadcare.php]: 2.6.0 still

**Comment by bautrukevich** [comment]
Fix plugin version

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,139 +1,172 @@
...
+The format is based now on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [2.6.0] — 2018-05-14
+
+### Added
+- Integration to the widget config to provide info on the Wordpress version and plugin version for User Agent with requests to Uploadcare Upload API.
+
+### Changed
+- Updated [uploadcare/uploadcare-php](https://github.com/uploadcare/uploadcare-php) to 2.2.0 version
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: 2.2.1

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,139 +1,172 @@
 # Changelog
+All notable changes to this project will be documented in this file.
 
-### 2.5.3 ([Download](http://downloads.wordpress.org/plugin/uploadcare.2.5.3.zip))
+The format is based now on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [2.6.0] — 2018-05-14
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]: update date and ship!

Comment (bautrukevich) [reply CHANGELOG.md]: ok!

**Comment by bautrukevich** [comment]
@dayton1987 I updated the date in the changelog and the readme please, look at this, and if it's all ok I will release a new version.

**Comment by bautrukevich** [comment]
[Выпустить новую версию библиотеки uploadcare-wordpress](https://trello.com/c/8vuZxeCu/131-%D0%B2%D1%8B%D0%BF%D1%83%D1%81%D1%82%D0%B8%D1%82%D1%8C-%D0%BD%D0%BE%D0%B2%D1%83%D1%8E-%D0%B2%D0%B5%D1%80%D1%81%D0%B8%D1%8E-%D0%B1%D0%B8%D0%B1%D0%BB%D0%B8%D0%BE%D1%82%D0%B5%D0%BA%D0%B8-uploadcare-wordpress)

**Pull request merged**